haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -548,3 +551,111 @@ managementSpec originate = do
+              ]
+
+          withSender commonOperator $ lCallEP stablecoinContract (Call @"Transfer") transfers
+
+          validate $ Left (lExpectAnyMichelsonFailed safelistContract)
+
+      it "can make mint operation fail" $ integrationalTestExpectation $ do
+        safelistContract <- lOriginate Safelist.safelistContract "Safelist test dummy" safelistStorage (unsafeMkMutez 0)
+        let
+          originationParams = addAccount (wallet1, (commonOperators, 10))
+              $ defaultOriginationParams {
+                  opPermissionsDescriptor = permissionDescriptorOperatorTransferAllowed,
+                  opSafelistContract = Just safelistContract
+                }
+        withOriginated originate originationParams $ \stablecoinContract -> do
+          let
+            transfers =
+              [ (#from_ .! wallet1, (#to_ .! wallet2, (#token_id .! 0, #amount .! 10)))
+              ]
+
+          withSender commonOperator $ lCallEP stablecoinContract (Call @"Transfer") transfers
+
+          validate $ Left (lExpectAnyMichelsonFailed safelistContract)
+
+      it "can make burn operation fail" $ integrationalTestExpectation $ do
+        safelistContract <- lOriginate Safelist.safelistContract "Safelist test dummy" safelistStorage (unsafeMkMutez 0)
+        let
+          originationParams =
+              addMinter (wallet1, 0)
+            $ addAccount (wallet1, (commonOperators, 35))
+            $ defaultOriginationParams { opSafelistContract = Just safelistContract }
+        withOriginated originate originationParams $ \stablecoinContract -> do
+
+          withSender wallet1 $ lCallEP stablecoinContract (Call @"Burn") [ 10 ]
+
+          validate $ Left (lExpectAnyMichelsonFailed safelistContract)
+
+    describe "Safelist contract interaction: approve behavior" $ do
+      let safelistStorage = Safelist.Storage
+            { sTransfers = [(wallet1, wallet2)]
+            , sReceivers = Set.fromList [wallet1, wallet2]
+            }
+      it "can approve transfers" $ integrationalTestExpectation $ do
+        safelistContract <- lOriginate Safelist.safelistContract "Safelist test dummy" safelistStorage (unsafeMkMutez 0)
+        let
+          originationParams = addAccount (wallet1, (commonOperators, 10))
+              $ defaultOriginationParams {
+                  opPermissionsDescriptor = permissionDescriptorOperatorTransferAllowed,
+                  opSafelistContract = Just safelistContract
+                }
+        withOriginated originate originationParams $ \stablecoinContract -> do
+          let
+            transfers =
+              [ (#from_ .! wallet1, (#to_ .! wallet2, (#token_id .! 0, #amount .! 10)))
+              ]
+
+          withSender commonOperator $ lCallEP stablecoinContract (Call @"Transfer") transfers
+
+          validate (Right expectAnySuccess)
+
+      it "can approve mint operation" $ integrationalTestExpectation $ do
```

Comment:
Same here, there's not actual minting happening

Reply:
Obviously missed to change it after copy pasting from above..

Reply:
Fixed.